src/main/java/me/moodcat/database/entities/Room.java

```diff
@@ -45,28 +51,53 @@
     private Integer id;
 
     /**
+     * The name of the room.
+     */
+    @Column(name = "roomName")
+    private String roomName;
+
+    @Embedded
+    @JsonIgnore
+    private VAVector valenceArousal;
+
+    /**
      * The current song of the room.
      */
     @ManyToOne
     @JoinColumn(name = "currentSong")
-    private Song song;
+    private Song currentSong;
```

Comment:
Maybe we should add the annotation `@JsonProperty(name = "song")`. I think when we store multiple types of songs in a room, we should be more specific about what type of song we actually mean.